code/modules/mob/living/emote.dm

```diff
@@ -107,9 +107,14 @@
 		message_animal_or_basic = custom_message
 	. = ..()
 	message_animal_or_basic = initial(message_animal_or_basic)
+	var/mob/living/carbon/carbon_user = user
+	if(!user.can_speak() || user.oxyloss >= 50)
```

Comment:
are you sure there isnt a define for this? this is trying to mimick the behavior of going unconcious at 50+ oxyloss, right?

Reply:
well not mimick, but use it

Reply:
This is a death gasp, if you're using it you're not even unconscious,  you're almost dead.
This is why it counts oxy damage, so make deathgasp play only if you died from any other damage

Reply:
> This is a death gasp, if you're using it you're not even unconscious, you're almost dead. This is why it counts oxy damage, so make deathgasp play only if you died from any other damage

fair, was a bit of a weird question i had anyway